### ISSUE TITLE
Please Check

### ISSUE THREAD

**Issue by samsiddiquiz** [malharsk27/Autoattendance-Cognitive#1]
![untitled](https://user-images.githubusercontent.com/32498803/31196468-bb3bf452-a902-11e7-85b7-82cc0df250ec.png)

**Comment by Abhishek-dev2** [comment]
Create this file(I don't know why it is missing) and then add the content from this [link](https://github.com/opencv/opencv/blob/master/data/haarcascades/haarcascade_frontalface_alt.xml).

**Comment by samsiddiquiz** [comment]
Its Already Created :/ 
![untitled](https://user-images.githubusercontent.com/32498803/31199456-47b02ce2-a90c-11e7-8f26-50598c49df64.png)

**Comment by samsiddiquiz** [comment]
can you provide Complete tuts, how to use this project ?

**Comment by Abhishek-dev2** [comment]
This is our old project, it is really bad. Please download the latest version made upon dlib and microsoft cognitive vision api. If you still want to make this project work then tutorial are [here](http://fewtutorials.bravesites.com/). And her YouTube channel is [here](https://www.youtube.com/channel/UCdhwz5WIZWZPjOWPpl3dDFA).

**Comment by samsiddiquiz** [comment]
http://fewtutorials.bravesites.com/FYP 
file missing Please Help :)

**Comment by samsiddiquiz** [comment]
Sorry For taking your time, I need hard :(
can you provide latest Face recognition attendance system?

**Comment by Abhishek-dev2** [comment]
Yes, it is old. It used to work but her youtube channel is still working, you can visit that.

**Comment by Abhishek-dev2** [comment]
This repository is the latest [repository](https://github.com/malharsk27/Autoattendance-Cognitive). I don't know how you got the old one.

**Comment by samsiddiquiz** [comment]
I dont know how to code and use python :/ But tried your this project but getting error please :)
![untitled](https://user-images.githubusercontent.com/32498803/31200529-2ac2b524-a910-11e7-9b71-bb63ed4bb636.png)

**Comment by Abhishek-dev2** [comment]
[This](http://docs.opencv.org/3.2.0/d5/de5/tutorial_py_setup_in_windows.html) will help. I think you are in too much hurry. This setup will take some time, cannot do it in an hour.

**Comment by Abhishek-dev2** [comment]
Also, import cv2 is required only for opening and closing the images. You can do that without openCV. Just search it on stack overflow.

**Comment by samsiddiquiz** [comment]
I'm Second Year Student, My Project Is Face Detection Attendance In Asp.net :/ but in internet i could not found any similar project :/

**Comment by Abhishek-dev2** [comment]
I learned the basic face detection from the documents of OpenCV which are the best, then I found dlib which had an accuracy of around 95% in face detection, far superior to OpenCV and implementing face detection using dlib and python was super easy. I did it in just 2 weeks(complete detection and recognition). I recommend dlib. Just read there documentations and you will also be able to implement it in just a week. But I am not sure about the language, last time I checked, dlib and OpenCV were available in c++ and python only. If you don't know python, then c++ is also very simple to use.

**Comment by samsiddiquiz** [comment]
Can you provide link ? where from you learned ? :)
an thank you so much for your Response

**Comment by samsiddiquiz** [comment]
Traceback (most recent call last):
  File "<pyshell#0>", line 1, in <module>
    import cv2
ImportError: DLL load failed: %1 is not a valid Win32 application.


i'm getting this error while importing cv2 in python idle.

**Comment by Abhishek-dev2** [comment]
Have you installed OpenCV's python version?? Almost all of the error's solutions are available on google. Links: [DLIB Documentation](http://dlib.net/python/index.html), [google search](https://www.google.co.in/search?q=dlib+face+detection+python&oq=dlib+face+dete&gs_l=psy-ab.3.1.0j0i20i263k1j0l8.3960.7891.0.9405.16.15.1.0.0.0.359.2841.0j6j4j2.12.0....0...1.1.64.psy-ab..3.13.2850...35i39k1j0i67k1j0i10k1j0i131k1j0i22i10i30k1j0i22i30k1.0.k8zVCcRqAtk), [opencv face recognition](http://docs.opencv.org/2.4/modules/contrib/doc/facerec/facerec_tutorial.html). This is the last reply as I am going to sleep.

**Comment by samsiddiquiz** [comment]
Bro Help :/ I Insntalle
![untitled](https://user-images.githubusercontent.com/32498803/31223792-514321f0-a980-11e7-9926-2e9718b2dc08.png)



RuntimeError: module compiled against API version 0xa but this version of numpy is 0x7

Traceback (most recent call last):
  File "<pyshell#2>", line 1, in <module>
    import cv2
ImportError: numpy.core.multiarray failed to import

**Comment by samsiddiquiz** [comment]
when success in importing cv2 getting error :/ 



![Uploading Untitled.png…]()

**Comment by Harsh1091996** [comment]
Hey can u plz help me out its urgent I'm not able to create a person it is showing error 404 person group not found plz help me...
